src/specs/RiveImage.nitro.ts

```diff
@@ -0,0 +1,28 @@
+import type { HybridObject } from 'react-native-nitro-modules';
+
+/**
+ * A Rive-compatible image that can be used for:
+ * - Referenced assets (out-of-band asset loading)
+ * - Image data binding (future: ViewModelImageProperty)
+ *
+ * The image stores the raw encoded bytes and is decoded when assigned to an asset.
+ */
+export interface RiveImage
+  extends HybridObject<{ ios: 'swift'; android: 'kotlin' }> {
+  /** The size of the image data in bytes */
+  readonly byteSize: number;
+}
+
+/**
+ * Factory for creating RiveImage instances from various sources.
+ * Exposed as `RiveImages` in the public API.
+ */
+export interface RiveImageFactory
```

Comment:
Should we add more options on to op URL. For .riv file loading we have network/local-assets/bundled/bytes. Might not need all the same options, or would we expect in React Native people will normally be working with a network asset?